doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -1,223 +1,275 @@
+Issue: When you run the {PRODUCT} installer, the installation path step references to {JWS} 5.2 instead of {JWS} {JWS_VERSION}.
+
+Workaround: None.
 
-Workaround: Deactivate the Reusable sub-process and activate it again.
+== DMN Designer
 
-.An error occurs when editing the process during restoring the other process [https://issues.redhat.com/browse/RHPAM-2757[RHPAM-2757]]
+.When you include a PMML 4.4 model into a DMN model and then save or validate the model, DMN throws an error [https://issues.redhat.com/browse/RHDM-1400[RHDM-1400]]
 
-Issue: In the process designer, While editing the process when you try to restore other process an error message is shown.
+Issue: You can not include a PMML 4.4 model into a DMN model, DMN model throws an error.
 
 Steps to reproduce:
 
-. Create process A and do not close this process.
-. Create process B.
-. Make a change in process B and click *Save*.
-. Click *Latest version* and select *Version 1*. Do not click *Restore*.
-. Open the process A.
-. Make a change in process A and click *Save*.
+* Create a new empty project in {CENTRAL}.
+* Create an empty DMN model.
+* Import one PMML asset with the 4.4 version into the project.
+* Open the DMN model.
+* Select the *Included Models* tab.
+* Click the *Include Model* and select the PMML 4.4 model.
+* Click *Include*.
 
-Expected result: No error messages are shown.
+Expected result: You can include a PMML 4.4 model into a DMN model.
 
-Actual result: An error message is shown.
+Actual result: You can not include a PMML 4.4 model into a DMN model.
 
-Workaround: Finish restoring the previous version by clicking *Restore* before you start editing other processes.
+Workaround: None
 
-.You cannot remove the case file and global variables [https://issues.redhat.com/browse/RHPAM-2643[RHPAM-2643]]
+.When you try to invoke a Business Knowledge Model (BKM) function from included DMN model, the validation fails [https://issues.redhat.com/browse/RHDM-1395[RHDM-1395]]
 
-Issue: When editing a case definition in process designer, you can add case file variables in the *Case Management* section, but you cannot delete them. It is also not possible to delete the global variables.
+Issue: You can not call the BKM function imported from another DMN model, which is aliased with the name containing `.` characters. If you try to invoke that function, the validation fails.
 
 Steps to reproduce:
 
-. Create a case project.
-. Create a case definition.
-. In the *Properties* panel add a case file variable in the *Case Management* section.
-. Add a global variable in the *Properties* panel.
-. Try to delete the case file variable and global variable.
+* Create an empty `Salary.dmn` DMN model.
+* Create an another empty `WorkingHours.dmn` DMN model.
+* Import `WorkingHours.dmn` DMN model into `Salary.dmn` DMN model.
+* Set an alias for `WorkingHours.dmn` model by including `.` character. For example, `working.hours.model`.
+* Validate the model. You receive an error message.
```

Comment:
I'd suggest using a numbered list instead of bullets since these are sequential steps.